dbt/models/shared_columns.md

```diff
@@ -797,10 +797,15 @@ percentage of fair cash value at which a property is assessed for taxing
 purposes. See `ccao.class_dict` for more information
 {% enddocs %}
 
+## is_active_exe_homeowner
```

Comment:
I'm not sure if these should be shared columns, but we don't have a `columns.md` in the model... model

Reply:
They can be shared, that's fine.